prereq_tools/base.py

```diff
@@ -785,6 +788,13 @@ def _setup_compiler(self):
         env = self.__env.Clone()
         config = Configure(env)
 
+        if compiler == 'covc':
+            covfile = self.__env.get('PREFIX') + "/../../test.cov"
```

Comment:
This doesn't look right...What directory are your trying to use?  The source dir?   This won't work if someone specifies PREFIX as, say, /usr/local

I think you probably want Dir('#').abspath which conveniently is already saved in self.__top_dir